spec/search-api.yaml

```diff
@@ -0,0 +1,119 @@
+---
+openapi: "3.0.0"
```

Comment:
Would it make sense to start with 3.0.2? I'm not sure what the patches are, just thinking it's probably easier to go with the latest right from the beginning.